src/app/tasks/[task_id]/page.tsx

```diff
@@ -5,6 +5,7 @@ import { MenuBar } from '@/components/elements/menu-bar';
 import { BreadcrumbComponent } from '@/components/elements/breadcrumb';
 import { BackButton } from '@/components/elements/backButton';
 import { DeleteTask } from '@/components/elements/deleteTask';
+import NavBar from '@/components/elements/nav-bar';
```

Comment:
ลบออก

Reply:
- [ ] deleted unused lines

- ToolTipProviders are essential for run code